governance/xc_admin/packages/xc_admin_frontend/components/tabs/General.tsx

```diff
@@ -301,9 +301,14 @@ const General = ({ proposerServerUrl }: { proposerServerUrl: string }) => {
           ? mapKey(multisigAuthority)
           : multisigAuthority
 
-        const initPublisher = async (publisherKey: PublicKey) => {
+        const initPublisherInPriceStore = async (publisherKey: PublicKey) => {
+          // Price store is only available in Pythnet and Pythtest-crosschain
+          if (cluster !== 'pythnet' && cluster !== 'pythtest-crosschain') {
```

Comment:
Maybe a better option would be to check if price store program is deployed on the current chain?

Reply:
Hmmm that also works. I wanted to avoid doing it for pythtest-conformance but it doesn't matter probably.